sesman/chansrv/Makefile.am

```diff
@@ -23,6 +23,11 @@ AM_CPPFLAGS += -DXRDP_FUSE $(FUSE_CFLAGS)
 CHANSRV_EXTRA_LIBS += $(FUSE_LIBS)
 endif
 
+if XRDP_FDK_AAC
+AM_CPPFLAGS += -DXRDP_FDK_AAC
+CHANSRV_EXTRA_LIBS += -lfdk-aac
```

Comment:
can be written like this with pkg-config
`CHANSRV_EXTRA_LIBS += $(FDKAAC_LIBS)`